### DISCUSSION TITLE
Not implemented Error: Enum value: '176'

### DISCUSSION THREAD

**Discussion by Incredible43-arch** [duckdb/duckdb#10968] [Q&A]
We encounter a problem of not implemented Error: Enum value: '176' not implemented. I want to trace this error, but the db system is so  complicated. Can u tell me how to locate the code causing this error? What tools can we use?

**Comment by Mause** [comment]
If you're wanting to debug this yourself, you can look at using lldb or similar? Otherwise, if you can provide us with an example that reproduces the issue, we'll look into it

**Comment by Tishj** [comment]
Sounds like a postgres expression supported by the parser, but not the transformer
If I would have to guess

**Comment by Incredible43-arch** [comment]
I try to implement a physical operator for comparison join, I imitated the implementation of nested loop join. When I insert the queries with comparison condition, some time it will give the correct result, but when I try this again, it will throw an error.
I run the db on shell. Here is return code:
![image](https://github.com/duckdb/duckdb/assets/78741638/b22b3896-89ea-45cf-bed0-4c4f4d4421bc)

**Reply by Tishj** [reply]
You need to regenerate enum_util, theres a script in `./scripts/`
